drink/src/session/record.rs

```diff
@@ -21,20 +21,23 @@ type ContractExecResult<R> = pallet_contracts::ContractExecResult<BalanceOf<R>,
 /// By `result` we mean the full result (enriched with some context information) of the contract
 /// interaction, like `ContractExecResult`. By `return` we mean the return value of the contract
 /// execution, like a value returned from a message or the address of a newly instantiated contract.
-pub struct Record<R: RuntimeWithContracts> {
+pub struct Record<Config: SandboxConfig>
```

Comment:
can we leave `Record` generic over `R: pallet_contracts::Config`? so that we don't introduce `Config: SandboxConfig` anywhere in this module

Reply:
done I think I could not do it because of the recording APIs I moved that to the session